vllm/executor/ray_gpu_executor.py

```diff
@@ -297,8 +304,10 @@ def execute_model(
         if self.forward_dag is None:
             self.forward_dag = self._compiled_ray_dag(enable_asyncio=False)
 
-        outputs = ray.get(self.forward_dag.execute(execute_model_req))
-        return outputs[0]
+        serialized_data = self.input_encoder.encode(execute_model_req)
+        outputs = ray.get(self.forward_dag.execute(serialized_data))
+        output = self.output_decoder.decode(outputs[0])
```

Comment:
Can we add a config guard and make the feature configurable?

Reply:
added an additional env var

Reply:
Looks like VLLM_SPMD_SEND_DELTA_DATA does not take effect right now?